elastalert/elastalert.py

```diff
@@ -1132,7 +1132,9 @@ def start(self):
             if self.args.end:
                 endtime = ts_to_dt(self.args.end)
 
-                if next_run.replace(tzinfo=dateutil.tz.tzutc()) > endtime:
+                dt = next_run.replace(tzinfo=dateutil.tz.tzutc())
+                if dt > endtime:
```

Comment:
change variable name from `dt` to `dtnow` to make it clearer that endtime is being compared to the current time?